tests/unit/mixins/data-adapter-mixin-test.js

```diff
@@ -123,6 +124,23 @@ describe('DataAdapterMixin', () => {
   });
 
   describe('#headersForRequest', function() {
+    it('shows a deprecation warning', function() {
+      let warnings;
+      registerDeprecationHandler((message, options, next) => {
+        // in case a deprecation is issued before a test is started
```

Comment:
I'm not sure what this comment means here since the `registerDeprecationHandler` is being used inside a test 🤔 

Also, wouldn't it be the same to do:
```
      let warnings = [];
      registerDeprecationHandler((message, options, next) => {
        warnings.push(message);
        next(message, options);
      });
```

Reply:
I think you're right, yes - I copied this from elsewhere and should have paid a bit more attention probably ;) - changed it now as you suggested